lib/reply.js

```diff
@@ -59,7 +61,7 @@ Reply.prototype.send = function (payload) {
     return
   }
 
-  if (payload instanceof Error) {
+  if (payload instanceof Error || this[isError] === true) {
```

Comment:
can you please do the `this[isError]` check only once and store it in a variable?

Reply:
Thanks, good catch. Done in 67e0ed5